PayPal/src/main/java/com/braintreepayments/api/PayPalPendingRequest.kt

```diff
@@ -0,0 +1,44 @@
+package com.braintreepayments.api
+
+import org.json.JSONObject
+
+/**
+ * A pending request for the PayPal web-based authentication flow created by invoking
+ * [PayPalLauncher.launch]. This pending request should be stored locally within the app or
+ * on-device and used to deliver a result of the browser flow in
+ * [PayPalLauncher.handleReturnToAppFromBrowser]
+ */
+sealed class PayPalPendingRequest {
+
+    /**
+     * A pending request was successfully started. This [PayPalPendingRequest.Started] should be
+     * stored and passed to [PayPalLauncher.handleReturnToAppFromBrowser]
+     */
+    class Started(val request: PayPalBrowserSwitchRequest) : PayPalPendingRequest() {
+        fun toJsonString(): String {
```

Comment:
Added these methods to make it easier for merchants to store the pending request as a String rather than having to deal with parcelizing/serializing/storing as an object. Open to other ideas for how to make this as merchant-friendly as possible.

Reply:
See comments in [this PR](https://github.com/braintree/browser-switch-android/pull/83/files) for alternative options for this